### ISSUE TITLE
Custom build command renders template before args

### ISSUE THREAD

**Issue by gbates101** [devspace-cloud/devspace#1019]
**What happened?**  
When specifying a custom build config for Google Cloud Build, the args are placed after the `[IMAGE]:[TAG]`, causing an invalid command to be rendered:

![image](https://user-images.githubusercontent.com/24995887/76874105-10140100-6845-11ea-8f49-da18590ae2c4.png)

If I forego the `args` section and instead place all of the args inline with the command, it includes the args into the binary name when it attempts to execute it:

![image](https://user-images.githubusercontent.com/24995887/76874983-4ef68680-6846-11ea-81ea-a2f5961bcb51.png)

**What did you expect to happen instead?**
Per the explanation section in the [docs](https://devspace.sh/cli/docs/configuration/images/custom#command), the `[IMAGE]:[TAG]` substitution should be after the args. If this is the intended behavior (i.e. docs are out of date), disregard this report.

**How can we reproduce the bug?**
Run `devspace build` with the following config in place:
```yaml
version: v1beta7
images:
  site:
    image: gcr.io/plaidcloud-build/site
    build:
      custom:
        command: gcloud
        args:
        - builds
        - submit
        - --tag
```

**Local Environment:**  
- DevSpace Version: 4.6.2
- Operating System: Windows (WSL)

**Anything else we need to know?**  
I think I can work around this with a script, but I'd rather not version a one-line script for each project we build and deploy using DevSpace.

/kind bug

**Comment by FabianKramm** [comment]
@gbates101 thanks for reporting this issue! Yes currently the behaviour is that args are placed after the image, I'll change this to reflect the docs and also add a new option `custom.afterArgs` to be able to specify arguments after the image tag(s)

**Comment by FabianKramm** [comment]
Should be fixed with v4.7.0
